### ISSUE TITLE
Add message to warn users when more than 255 non-ESL plugins or 4096 ESL plugins are active

### ISSUE THREAD

**Issue by sibir-ine** [loot/loot#1837]
We currently log this in the debug log, but it'd be helpful to point this out to users more explicitly. Also, the fact that we display plugins past these limits as inactive causes confusion, so we should explicitly state that LOOT is doing this as well as why (consistency with the game's behavior) in the message.

**Comment by Ortham** [comment]
The log message is actually a bit misleading, it's actually only relevant when using Morrowind with MWSE: https://github.com/loot/loot/blob/master/src/gui/state/game/game.cpp#L720-L733.

There are three conflicting goals at work:

1. LOOT tries to interpret the load order as the game does. This means that LOOT considers any active plugins past the limit to be inactive.
2. LOOT shouldn't write invalid load orders. This means that any active plugins past the limit will be deactivated when the load order is written.
3. LOOT shouldn't make changes to what plugins are active. There's no problem achieving this when the number of active plugins is under or at the limit. If there are too many plugins active, LOOT prioritises not writing an invalid load order.

I don't really want to change the behaviour, but adding a message explaining it would be a good idea. I think I'll need to add a method to libloadorder/libloot to check when this has happened, as I don't think it's exposed at the moment.

**Comment by Ortham** [comment]
I've realised that a problem with the current approach is that deactivating any plugins past the limit means that if the order of plugins is changed that could result in plugins that the game would load after sorting get deactivated as part of writing the sorted load order. As such, I'm removing that behaviour, so LOOT (actually libloadorder) won't deactivate any plugins on load.

I've also fixed up the message that's displayed when MWSE is installed, and added two new messages for when there are too many normal or light active plugins. However, these messages won't be displayed until libloot is updated to not deactivate excess plugins, since they only show when too many are active.
